### PR TITLE
Script for defect 1918

### PR DESCRIPTION
**Pull request by mkorniichuk**
ATF Test Scripts to check [#1918](https://github.com/SmartDeviceLink/sdl_core/issues/1918)

This PR is **not ready** for review.

### Summary
A test script to verify that issue is fixed

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

**Comment by dboltovskyi** [comment]
Livio team please do not review this PR for now since it hasn't passed our internal review yet.  
Review and appropriate updates going be done till 1st of November.
Thank you.

**Comment by GetmanetsIrina** [comment]
PR is rebased on current develop(a2040db) branch.
Script is applicable for all policy flows and transport types.
Script has been moved to 6_1 folder.
Test set for script is added.

File: test_scripts/Defects/6_1/1918_Multiple_apps_activation_during_active_embedded_audio_source.lua [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+  end)
+end
+
+local function deactivateApp(pAppId)
+  if not pAppId then pAppId = 1 end
+  common.getHMIConnection():SendNotification("BasicCommunication.OnAppDeactivated", { appID = common.getHMIAppId(pAppId) })
+  common.getMobileSession(pAppId):ExpectNotification("OnHMIStatus")
+  :ValidIf(function(_, data)
+    if config["application"..pAppId].registerAppInterfaceParams.isMediaApplication == true
+      then
```
Comment (jacobkeeler) [inline test_scripts/Defects/6_1/1918_Multiple_apps_activation_during_active_embedded_audio_source.lua:75]: `NAVIGATION` app would be `STREAMABLE`

**Review by jacobkeeler — CHANGES_REQUESTED** [review]

File: test_scripts/Defects/6_1/1918_Multiple_apps_activation_during_active_embedded_audio_source.lua [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+local function embededAudioActivated()
+  common.getHMIConnection():SendNotification("BasicCommunication.OnEventChanged",
+    { eventName = "AUDIO_SOURCE", isActive = true })
+  common.getMobileSession(1):ExpectNotification("OnHMIStatus", {
+    hmiLevel = "BACKGROUND", audioStreamingState = "NOT_AUDIBLE", videoStreamingState = "NOT_STREAMABLE"
+  })
+  common.getMobileSession(2):ExpectNotification("OnHMIStatus", {
+    hmiLevel = "LIMITED", audioStreamingState = "AUDIBLE", videoStreamingState = "NOT_STREAMABLE"
+  })
```
Comment (jacobkeeler) [inline test_scripts/Defects/6_1/1918_Multiple_apps_activation_during_active_embedded_audio_source.lua:97]:
```suggestion
  -- hmiLevel = "LIMITED", audioStreamingState = "AUDIBLE", videoStreamingState = "STREAMABLE"
  common.getMobileSession(2):ExpectNotification("OnHMIStatus"):Times(0)
```
App would be in `LIMITED`, `AUDIBLE`, `STREAMABLE`

File: test_scripts/Defects/6_1/1918_Multiple_apps_activation_during_active_embedded_audio_source.lua [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+      else
+        return data.payload.audioStreamingState == "NOT_AUDIBLE" and
+          data.payload.videoStreamingState == "NOT_STREAMABLE" and
+          data.payload.hmiLevel == "BACKGROUND" and
+          data.payload.systemContext == "MAIN"
+    end
+  end)
+end
+
+local function embededAudioActivated()
```
Comment (jacobkeeler) [inline test_scripts/Defects/6_1/1918_Multiple_apps_activation_during_active_embedded_audio_source.lua:89]:
```suggestion
local function embeddedAudioActivated()
```

File: test_scripts/Defects/6_1/1918_Multiple_apps_activation_during_active_embedded_audio_source.lua [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+    { hmiLevel = "FULL", audioStreamingState = "AUDIBLE" })
+  :Times(2)
+  :Do(function(exp)
+    if exp.occurences == 1 then
+      local requestId = common.getHMIConnection():SendRequest("SDL.ActivateApp", { appID = common.getHMIAppId(1) })
+      EXPECT_HMIRESPONSE(requestId)
+    end
+  end)
+  common.getMobileSession(2):ExpectNotification("OnHMIStatus",
+    { hmiLevel = "LIMITED", audioStreamingState = "AUDIBLE" })
```
Comment (jacobkeeler) [inline test_scripts/Defects/6_1/1918_Multiple_apps_activation_during_active_embedded_audio_source.lua:121]:
```suggestion
  -- hmiLevel = "LIMITED", audioStreamingState = "AUDIBLE", videoStreamingState = "STREAMABLE"
  common.getMobileSession(2):ExpectNotification("OnHMIStatus"):Times(0)
```
App would be in `LIMITED`, `AUDIBLE`, `STREAMABLE`

**Review by jacobkeeler — CHANGES_REQUESTED** [review]

**Comment by jacobkeeler** [comment]
Closing in favor of #2452

**Pull request closed without merging**